lib/alpha_taxonomy/taxon_link_deleter.rb

```diff
@@ -0,0 +1,27 @@
+module AlphaTaxonomy
+  class TaxonLinkDeleter
+    def initialize(logger: Logger.new(STDOUT))
+      @log = logger
+    end
+
+    def run!
+      base_paths = ENV.fetch("TAXON_CLEANUP").split(",")
+
+      base_paths.each do |base_path|
+        @log.info "Clearing links for #{base_path}"
+
+        linked_items = Services.live_content_store.incoming_links!(
+          base_path, types: ['alpha_taxons']
+        )['alpha_taxons']
+
+        linked_items.each do |linked_item|
```

Comment:
`alpha_taxons` can be nil if the item is not tagged (anymore) to a taxon. I think this `each` would error in that case.